config/locales/en.yml

```diff
@@ -28,6 +28,24 @@ en:
     cohort_does_not_accept_capping: "Leave the '#/funded_place' field blank. It's only needed for participants starting NPQs from autumn 2024 onwards."
     cannot_change_funded_place: "You must void or claw back your declarations for this participant before being able to set '#/funded_place' to false"
 
+  declaration: &declaration
+    future_declaration_date: "The '#/declaration_date' value cannot be a future date. Check the date and try again."
+    mismatch_declaration_type_for_schedule: "The property '#/declaration_type' does not exist for this schedule."
+    participant_id:
+      blank: "The property '#/participant_id' must be present"
+      declaration_must_be_before_withdrawal_date: "This participant withdrew from this course on %{withdrawal_date}. Enter a '#/declaration_date' that's on or before the withdrawal date."
+      invalid_participant: "Your update cannot be made as the '#/participant_id' is not recognised. Check participant details and try again."
+    declaration_date:
+      blank: "Enter a '#/declaration_date'."
+      invalid: "Enter a valid RCF3339 '#/declaration_date'."
+      declaration_before_schedule_start: "Enter a '#/declaration_date' that's on or after the schedule start."
+    declaration_type:
+      blank: "Enter a '#/declaration_type'."
+    has_passed:
+      invalid: "Enter 'true' or 'false' in the '#/has_passed' field to indicate whether this participant has passed or failed their course."
+    course_identifier:
+      invalid_course: "The entered '#/course_identifier' is not recognised for the given participant. Check details and try again."
```

Comment:
same for this one I think it needs to move under the declaration/create as coming back as missing